### PR TITLE
Merge edit workflow modals to one

### PR DESCRIPTION
**Pull request by rvsia**
Part of https://projects.engineering.redhat.com/browse/SSP-1723

> 6. Consolidate all "edits" in the kebab dropdown into "edit"

There are no mockups, so I just did the add form with initials values + sequence input.

**After**

image

### CODE REVIEW

**Comment by rvsia** [comment]
@lgalis @Hyperkid123

**Comment by Hyperkid123** [comment]
cc @sbuenafe-rh @fionayhlin

File: src/smart-components/workflow/edit-workflow-modal.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      { !isLoading && <FormRenderer
+        onSubmit={ onSave }
+        onCancel={ onCancel }
+        schema={ schema }
+        initialValues={ initialValues }
+        FormTemplate={ (props) => <FormTemplate
+          { ...props }
+          submitLabel={ intl.formatMessage(commonMessages.save) }
+          buttonClassName="pf-u-mt-0"
+          disableSubmit={ [ 'validating' ] }
```
Comment (Hyperkid123) [inline src/smart-components/workflow/edit-workflow-modal.js:87]: Edit should also have disabled submit button when 'pristine'.

**Comment by fionayhlin** [comment]
@rvsia @Hyperkid123 Looks good! 👍

**Pull request merged**